src/index.html

```diff
@@ -5,8 +5,383 @@
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">
+              Reviews: 5
+            </p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-title">
+              Price:
+            </p>
+            <p class="card__price-item">
+              <span>&#36;</span>2,199
+            </p>
+          </div>
+          <a href="#buy" class="card__btn">
+            Buy
+          </a>
+        </div>
+        <!-- ***************************************************** -->
+        <div class="catalog__card card">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+            class="card__picture"
+          >
+          <p class="card__description">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="card__code">
+            Product code: 195434
+          </p>
+          <div class="card__rating">
+            <div class="card__stars stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card-reviews">
+              Reviews: 5
+            </p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-title">
+              Price:
+            </p>
+            <p class="card__price-item">
+              <span>&#36;</span>2,199
+            </p>
+          </div>
+          <a href="#buy" class="card__btn">
+            Buy
+          </a>
+        </div>
+        <!-- ***************************************************** -->
+        <div class="catalog__card card">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+            class="card__picture"
+          >
+          <p class="card__description">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="card__code">
+            Product code: 195434
+          </p>
+          <div class="card__rating">
+            <div class="card__stars stars stars--4">
+              <div class=" stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">
+              Reviews: 5
+            </p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-title">
+              Price:
+            </p>
+            <p class="card__price-item">
+              <span>&#36;</span>2,199
```

Comment:
Why do you need an additional `span` here?

Reply:
Symbol dollar

Reply:
Yep, I see. But you can print dollar sign without code and span usage:
```
<p class="card__price-item">
  $2,199
</p>
```